py/ig_compress_equal_clusters.py

```diff
@@ -6,11 +6,53 @@
 import os
 import os.path
 import tempfile
+from collections import defaultdict
 
 
-from ig_remove_low_abundance_reads import smart_open, parse_abundance
+from ig_remove_low_abundance_reads import smart_open
 
 
+def parse_cluster_mult(id):
+    import re
+    id = str(id)
+    m = re.match(r"^cluster___(\d+)___size___(\d+)$", id)
+    if m:
+        g = m.groups()
+        cluster = int(g[0])
+        mult = int(g[1])
+        return cluster, mult
+    else:
+        return None
+
+
+def check_fa_rcm_consistency(fa_filename, rcm_filename):
+    from collections import defaultdict
+
+    cluster_mult_rcm = defaultdict(int)
+    num_rcm_reads = 0
+    with open(rcm_filename) as rcm:
+        for line in rcm:
+            cluster = int(line.split("\t")[1])
```

Comment:
Isn't it number of the read in fa file, not the read index in its id?
I mean in line 42 cycle shouldn't you track record number instead of using parsed cluster number?